simbad/rotsearch/phaser_search.py

```diff
@@ -255,6 +261,23 @@ def summarize(self, csv_file):
         summarize_result(self.search_results,
                          csv_file=csv_file, columns=columns)
 
+    @staticmethod
+    def calculate_ermsd(nres, id):
+        """Calculate the estimated rmsd based on the number of residues and sequence id"""
+
+        if id < 0 or id > 100:
+            msg = "Sequence id must be a number between 0 and 100"
+            raise RuntimeError(msg)
+
+        if nres < 0:
+            msg = "The number of residues must be greater than 0"
+            raise RuntimeError(msg)
+
+        return np.around((1.54967686e+00 + 1.69034597e-03 * nres + -2.57055256e-02 * id + -5.92896517e-07 * nres ** 2
+                          + -1.68255006e-05 * nres * id + 2.03689221e-04 * id ** 2 + 1.01170096e-10 * nres ** 3
+                          + 2.45170962e-09 * nres ** 2 * id + 5.10824965e-08 * nres * id ** 2
+                          + -7.01029519e-07 * id ** 3), decimals=3)
```

Comment:
Wouldn't round the value to a specific number of decimals, users could do that or we can do it when printing the output, but you're not allowing anyone to access the full value